libs/wire-api-federation/src/Wire/API/Federation/API/Galley.hs

```diff
@@ -19,40 +19,63 @@
 
 module Wire.API.Federation.API.Galley where
 
+import Data.Aeson (FromJSON, ToJSON)
+import Data.Id (ConvId, UserId)
+import Data.Qualified (Qualified)
+import Imports
 import Servant.API (JSON, Post, ReqBody, (:>))
 import Servant.API.Generic ((:-))
+import Wire.API.Arbitrary (Arbitrary, GenericUniform (..))
+import Wire.API.Conversation (Conversation)
+import Wire.API.Federation.Util.Aeson (CustomEncoded (CustomEncoded))
+
+-- FUTUREWORK: data types, json instances, more endpoints. See
+-- https://wearezeta.atlassian.net/wiki/spaces/CORE/pages/356090113/Federation+Galley+Conversation+API
+-- for the current list we need.
 
 data Api routes = Api
-  { getConversation ::
+  { getConversations ::
       routes
         :- "federation"
         :> "conversations"
-        -- usecases:
-        -- - e.g. upon registering a new client to your account, get the list of your conversations
-        :> "list-conversations"
-        :> ReqBody '[JSON] ListConversations
-        :> Post '[JSON] ListConversationsResponse,
-    conversationMemberChange ::
+        :> "get-by-ids"
+        :> ReqBody '[JSON] GetConversationsRequest
+        :> Post '[JSON] [Conversation],
```

Comment:
Could we wrap the plain list `[Conversation]` here in a json object (e.g. `data SomeResponse = { conversations :: [Conversation] }` for better futureproofing?